src/engine/bank04.asm

```diff
@@ -1180,8 +1180,48 @@ Func_12871: ; 12871 (4:6871)
 Func_1288c: ; 1288c (4:688c)
 	INCROM $1288c, $128a9
 
-Func_128a9: ; 128a9 (4:68a9)
-	INCROM $128a9, $1296e
+OnPlayerNamingScreen:: ; 128a9 (4:68a9)
```

Comment:
Maybe ``DisplayPlayerNamingScreen`` would be more appropriate for a function name?``OnPlayerNamingScreen`` sounds more like a wram flag name to me.